packages/fiori/test/samples/Login.sample.html

```diff
@@ -0,0 +1,139 @@
+<header class="component-heading">
+    <h2 class="control-header">Login Component</h2>
+    <div class="component-heading-since">
+        <span><!--since_tag_marker--></span>
+    </div>
+</header>
+
+<div class="component-package">@ui5/webcomponents-fiori</div>
+
+<div class="control-tag">&lt;ui5-li-notification-group&gt;</div>
+
+<style>
+    body {
+        margin: 20px;
+        background-color: var(--sapBackgroundColor);
+    }
+
+    .ui5-showcase {
+        width: fit-content;
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        gap: 25px;
+    }
+
+    .item {
+        /* was 375px */
+        flex: 1 1 275px;
+    }
+</style>
+
+<div class="ui5-showcase">
+    <!-- Example WITH Header and Subheader -->
+    <div class="item">

```

Comment:
Just for your info - The Samples are the pages that users see and here we follow the pattern

```html
<section>
	<h3>Basic Login</h3>
	<div class="snippet">
		<ui5-login></ui5-login>
	</div>
	<pre class="prettyprint lang-html"><xmp>
   <ui5-login></ui5-login>
>
	</xmp></pre>
</section>

```